pkg/kademlia/kademlia.go

```diff
@@ -27,8 +27,6 @@ import (
 var (
 	// NodeErr is the class for all errors pertaining to node operations
 	NodeErr = errs.Class("node error")
-	// BootstrapErr is the class for all errors pertaining to bootstrapping a node
-	BootstrapErr = errs.Class("bootstrap node error")
```

Comment:
Same than before, have you checked if the issues is that the error has not been used where it should be rather than it exists without reason?